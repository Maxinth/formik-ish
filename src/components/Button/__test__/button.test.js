import { Button } from "../index";
import renderer from "react-test-renderer";

it("renders a snapshot of <Button />", () => {
  const btn = renderer.create(<Button />).toJSON();
  expect(btn).toMatchSnapshot();
});
