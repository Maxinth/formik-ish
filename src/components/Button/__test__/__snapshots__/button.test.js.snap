// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot of <Button /> 1`] = `
<button
  className="ant-btn ant-btn-default sc-bdvvtL ezlRpd"
  onClick={[Function]}
  type="button"
/>
`;
